### PR TITLE
f-content-cards@2.2.0.beta.5 - removes tenant specific styles and updates the styles to match design

### PR DESCRIPTION
**Pull request by matthewhardern**
Makes some minor style changes to remove the tenant specific styles, plus adds some style changes ro make it up to date with latest design.

### CODE REVIEW

File: packages/f-content-cards/src/components/cardTemplates/PromotionCard.vue [hunk 1]
```diff
@@ -3,16 +3,18 @@
         :card="card"
         :container-title="containerTitle"
         :is-carousel="isCarousel"
-        :data-test-id="testId">
+        :data-test-id="testId"
+        :bold-title="true"
+    >
         <span
             v-if="type === 'Promotion_Card_1'"
-            class="o-btn o-btn--secondary"
```
Comment (matthewhardern) [inline packages/f-content-cards/src/components/cardTemplates/PromotionCard.vue:9]: I have removed this a temp work around until we come up with an idea of what is the best way to have a button thats not a button here as the button styles are being removed, so for the time being I have created a temp style until we decide what to do

File: packages/f-content-cards/src/components/cardTemplates/FirstTimeCustomerCard.vue [hunk 2]
```diff
@@ -1,30 +1,21 @@
...
             </span>
-            <template v-if="!bannerBeforeDescription">
-                for first time user
-            </template>
         </template>
         <p
-            :class="['c-restaurantCard-footer', $style['c-restaurantCard-footer'], {
-                [$style['c-restaurantCard-footer-legacy']]: usesLegacyStyles
-            }]">
+            :class="['c-restaurantCard-footer', $style['c-restaurantCard-footer']]">
```
Comment (minrwhite) [inline packages/f-content-cards/src/components/cardTemplates/FirstTimeCustomerCard.vue:18]: Please remove the `-legacy` styles from the `<style>` section if we don't need them. Or are we keeping them for another purpose?

Comment (matthewhardern) [reply packages/f-content-cards/src/components/cardTemplates/FirstTimeCustomerCard.vue:18]: have removed

**Review by minrwhite — APPROVED** [review]

File: packages/f-content-cards/src/components/cardTemplates/FirstTimeCustomerCard.vue [hunk 3]
```diff
@@ -1,30 +1,21 @@
 <template>
     <card-container
         :card="card"
-        :banner-before-description="bannerBeforeDescription"
-        :bold-title="!usesLegacyStyles"
+        :banner-before-description="true"
```
Comment (ashleynolan) [inline packages/f-content-cards/src/components/cardTemplates/FirstTimeCustomerCard.vue:4]: Doesn't need to be done as part of this ticket, but could we update any boolean props to make sure they stick to `is/has/should` style naming. Helps when scanning files

Comment (ashleynolan) [reply packages/f-content-cards/src/components/cardTemplates/FirstTimeCustomerCard.vue:4]: Same for the `bold-title` prop below too

Comment (matthewhardern) [reply packages/f-content-cards/src/components/cardTemplates/FirstTimeCustomerCard.vue:4]: I was unaware, happy to open another PR at later date to modify

**Review by kevinrodrigues — APPROVED** [review]

File: packages/f-content-cards/src/components/cardTemplates/PromotionCard.vue [hunk 4]
```diff
@@ -3,16 +3,18 @@
...
+            :class="[$style['c-contentCard-link-promo-1']]"
```
Comment (ashleynolan) [inline packages/f-content-cards/src/components/cardTemplates/PromotionCard.vue:11]: Naming scheme thing 😆 

So not sure on xact hierarchy of classes here, but dashes are only used to indicate child elements of another classname. CamelCase is used for descriptors.

So in this instance, is there a `c-contentCard-link` and a `c-contentCard-link-promo` class? (as this is indicating that `c-contentCard-link-promo-1` is the child element of those classes, or should this be `c-contentCard-linkPromo1`

File: packages/f-content-cards/src/components/cardTemplates/PromotionCard.vue [hunk 5]
```diff
@@ -3,16 +3,18 @@
...
             :data-test-id="testIdForPromoCardType(1)">
             {{ ctaText }}
         </span>
         <span
             v-if="type === 'Promotion_Card_2'"
-            :class="[$style['c-contentCard-link']]"
+            :class="[$style['c-contentCard-link-promo-2']]"
```
Comment (ashleynolan) [inline packages/f-content-cards/src/components/cardTemplates/PromotionCard.vue:17]: Naming as above

File: packages/f-content-cards/src/components/cardTemplates/FirstTimeCustomerCard.vue (hunk 3 above)
Comment (minrwhite) [reply packages/f-content-cards/src/components/cardTemplates/FirstTimeCustomerCard.vue:4]: I could look at doing this as part of my PR to merge in the HPC hotfixes, if you'd like @matthewhardern ?

File: packages/f-content-cards/src/components/cardTemplates/voucherCard/VoucherCard.vue [hunk 6]
```diff
@@ -3,7 +3,9 @@
         :card="card"
         :cta-enabled="false"
         :isolate-hero-image="isAnniversaryCard"
-        :embolden-text="isAnniversaryCard">
+        :embolden-text="isAnniversaryCard"
```
Comment (ashleynolan) [inline packages/f-content-cards/src/components/cardTemplates/voucherCard/VoucherCard.vue:6]: Same for these props as above. Happy it being changed in a separate PR though.

File: packages/f-content-cards/src/components/cardTemplates/FirstTimeCustomerCard.vue [hunk 7]
```diff
@@ -1,30 +1,21 @@
...
+        :bold-title="true"
```
Comment (DamianMullins) [inline packages/f-content-cards/src/components/cardTemplates/FirstTimeCustomerCard.vue:5]: If you're setting the values to `true` you can define the prop as just `bold-title`.

**Pull request merged**